File: ACM/Python/WFUStudy/新学期新气象/tempCodeRunnerFile.py
'''
Author: KasperFan && fanwlx@foxmail.com
Date: 2024-03-10 15:04:32
LastEditTime: 2024-03-10 15:35:56
FilePath: /Python/WFUStudy/新学期新气象/P1048.py
describes: This file is created for learning Python to deal OJ problems.
Copyright (c) 2024 by KasperFan in WFU, All Rights Reserved. 
'''
import numpy as np
from array import array
from sys import stdin
from io import StringIO

input = lambda: stdin.readline().strip()
buffer = StringIO()


class Perl():
    def __init__(self, time , value) -> None:
        self.time, self.value = time, value
        pass

T, M = map(int, input().split())
dp = array('i', [0]*(T+1))
perls = [Perl(*map(int, input().split())) for _ in range(M)]
for i in range(M):
    for j in range(perls[i].time, T+1):
        dp[j] = max(dp[j], dp[j-perls[i].time]+perls[i].value)
print(dp[T])

File: Learn/C/Data_Structure/第1章_概论/1.1_引子/tempCodeRunnerFile.c
/*
 * @Author: KasperFan && fanwlx@foxmail.com
 * @Date: 2023-11-28 23:51:15
 * @LastEditTime: 2024-03-08 14:24:48
 * @FilePath: /C/Data_Structure/第1章_概论/1.1_引子/tempCodeRunnerFile.c
 * @describes: This file is created for learning Code.
 * Copyright (c) 2024 by KasperFan in WFU, All Rights Reserved.
 */
// void PrintN(int N)
// { /* 打印从1到N的全部正整数 */
//     if (N > 0)
//     {
//         PrintN(N - 1);
//         printf("%d\n", N);
//     }
// }

#include <stdio.h>
#include <string.h>
#include <time.h>
// #include <windows.h>
#include <unistd.h>

int main() {
    clock_t start = clock();
    char str[] = "Hello,World!";
    for (int i = 0; str[i]; i++)
    {
        printf("%c", str[i]);
        // fflush(stdout);
        usleep(70000);
    }
    printf("\n");
    return 0;
}
